installation-bundle/src/Controller/InstallationController.php

```diff
@@ -459,7 +459,9 @@ private function importExampleWebsite()
             $installTool->persistConfig('exampleWebsite', null);
             $installTool->logException($e);
```

Comment:
Why are you not logging the root exception here as well?

Reply:
I didn’t want to overwhelm users with a too verbose error message in the install tool. That’s why I only show the message of the root exception (mostly a simple SQL error message).

But in the log a more verbose message makes sense IMO so I wouldn’t change that here.